ds3-autogen-go/src/main/kotlin/com/spectralogic/ds3autogen/go/generators/request/BaseRequestGenerator.kt

```diff
@@ -48,7 +50,28 @@ open class BaseRequestGenerator : RequestModelGenerator<Request>, RequestModelGe
                 structParams,
                 withConstructors,
                 nullableWithConstructors,
-                voidWithConstructors)
+                voidWithConstructors,
+                imports)
+    }
+
+    /**
+     * Retrieves imports that are not present in all request files
+     */
+     override fun toImportSet(ds3Request: Ds3Request): ImmutableSet<String> {
+        if (importStrconv(ds3Request.requiredQueryParams) || importStrconv(ds3Request.optionalQueryParams)) {
+            return ImmutableSet.of("strconv")
+        }
+        return ImmutableSet.of()
+    }
+
+    /**
+     * Checks if strconv should be imported based on the types of the parameters
+     */
+    fun importStrconv(ds3Params: ImmutableList<Ds3Param>?): Boolean {
```

Comment:
You should probably rename this so that way the method name is more in the form of a question like, `isStrconvImportRequired` or something along those lines.